Automattic/jetpack
# Instagram Latest Posts: clarify use of access token id

#### Changes proposed in this Pull Request:

Clarifies that the access token used for fetching posts from Instagram is actually a numeric id.

Syncs the changes from D81330-code

#### Other information:


#### Jetpack product discussion

n/a

#### Does this pull request change what data or activity we track or use?

no

#### Testing instructions:

* Insert an Instagram Latest Posts block in the editor and connect to Instagram
* Save the post and make sure you can see the images from your feed on the front end of the site (check both logged in and logged out views)
